com.avaloq.tools.ddk.xtext.export/src/com/avaloq/tools/ddk/xtext/export/generator/FingerprintComputerGenerator.xtend

```diff
@@ -36,9 +36,10 @@ class FingerprintComputerGenerator {
       package «fingerprintComputer.toJavaPackage»;
 
       import org.eclipse.emf.ecore.EObject;
+      «IF !interfaces.isEmpty»
       import org.eclipse.emf.ecore.EPackage;
       import org.eclipse.emf.ecore.util.Switch;

```

Comment:
Don't we need `org.eclipse.emf.ecore.util.Switch` unconditionally at line 61?

Reply:
I do not think so, because if the list is empty, the for loop will have zero iterations.